.github/workflows/lint.yml

```diff
@@ -0,0 +1,42 @@
+name: Lint
+
+on: push
+
+jobs:
+  run-linters:
+    name: Run linters
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Git repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # all history needed for commitlint to work
+
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+
+      - name: Install Node dependencies
+        run: npm ci --no-audit --no-optional
+
+      - name: Run commitlint
+        uses: wagoid/commitlint-github-action@v2
```

Comment:
Not entirely sure @rabelloo if using a predefined action here is any better, what do you think?

Reply:
I don't mind either, as long as it accomplishes the goal, which is:
1. to lint all commit messages being added and
2. verbosely explain why it passed or failed